ui/lib/components/CopyLink/index.tsx

```diff
@@ -43,15 +58,20 @@ function CopyLink({ data }: ICopyLinkProps) {
     reset()
   }
 
+  const copyBtn = (
+    <CopyToClipboard text={data} onCopy={handleCopy}>
+      <a>
+        {t(`component.copyLink.${transKeys[displayVariant]}`)} <CopyOutlined />
+      </a>
+    </CopyToClipboard>
+  )
+
   return (
     <span>
-      {!showCopied && (
-        <CopyToClipboard text={data} onCopy={handleCopy}>
-          <a>
-            {t('component.copyLink.text')} <CopyOutlined />
-          </a>
-        </CopyToClipboard>
+      {!showCopied && displayVariant === 'original_sql' && (
+        <Tooltip title={t('component.copyLink.tooltip')}>{copyBtn}</Tooltip>
```

Comment:
Maybe here is a mistake. The tooltip only exists specifically for "original_sql" but not "formatted_sql". Maybe you can just remove this tooltip.

Reply:
Done!